src/mame/sinclair/spectrum.cpp

```diff
@@ -355,7 +355,11 @@ void spectrum_state::spectrum_ula_w(offs_t offset, uint8_t data)
 	if ((Changed & (1<<4))!=0) m_speaker->level_w(BIT(data, 4));
 
 	/* write cassette data */
-	if ((Changed & (1<<3))!=0) m_cassette->output((data & (1<<3)) ? -1.0 : +1.0);
+	if ((Changed & (1<<3))!=0)
+	{
+		m_speaker->level_w(BIT(data, 3));
+		m_cassette->output((data & (1<<3)) ? -1.0 : +1.0);
+	}
```

Comment:
Why not use `BIT(x, 3)` in all three cases?  Like:
```
	if (BIT(Changed, 3))
	{
		m_speaker->level_w(BIT(data, 3));
		m_cassette->output(BIT(data, 3)) ? -1.0 : +1.0);
	}
```

Also, for consistency, `Changed` should really be `changed` (snake case) as it’s a local, not a template parameter.